### ISSUE TITLE
[req] show program running status, not just window count (re: removal of sys tray)

### ISSUE THREAD

**Issue by benjamindaines** [micheleg/dash-to-dock#565]
With the system tray being dropped in gnome 3.26 there is no longer any indication of a program continuing to run in the background, after its last window has been closed.  

Perhaps dash-to-dock should display a dim indicator on launchers that have not quit after all windows are closed, and the "quit" option in d2d's menu should terminate the background process.  

Programs where this is an issue: Discord, Slack, Cantata(qt).  None of these really target gnome, so I can't really see them being to quick to change their behavior.  

Having dash to dock reflect proper running indication would provide a pragmatic solution, as well as offering more complete functionality.

**Comment by micheleg** [comment]
Hi,

I undertand your point expecially now that the legacyTray has been removed alltogether and without anadditional extension there's no way to know if the application is running or not. That said, I'm a bit reluctant on including this functionality. I could:
1. Just manage to check on a tray icon to consider the running status of an application, adding or updating the respective launcher. For certain applicatoin this could feel natural (basically those that are just minimized to the tray), However
    * How easy it is to associate the try icon with the applciation launcher?
    * The behaviour on click would be quite incosistend: some application can raise their own window (e.g. slack), others don't  (e.g. Dropbox)
    * There would still be the need to access the tray icon menu for certain applications (e.g. Dropbox). 
   * Screen/Worskpace isolation: where does the application belong?
2. Integrate a tray area in the dock
   * I don't think I would  want to do that, as the dock would become a real Windows-like panel: in such case, I would recommend the use of the [dash-to-panel extension](https://extensions.gnome.org/extension/1160/dash-to-panel/)

The only thing that I see myself playing with at the moement is considering to update the running status of those applications that are pinned to the dock (which are most likely always able to launch a window) and are the at the moment  give a false information (is my slack client runnning or not?). But I can't promise anything.

**Comment by benjamindaines** [comment]
I agree with both your concerns, legacyTray is gone and taking on that functionality doesn't seem like a good idea.  I like the thought of restricting it to just the pinned launchers as well.
